include/fastdds/dds/publisher/DataWriter.hpp

```diff
@@ -214,6 +214,9 @@ class DataWriter : public DomainEntity
 
     RTPS_DllAPI ReturnCode_t assert_liveliness();
 
+    RTPS_DllAPI bool remove_all_changes(
```

Comment:
Return `ReturnCode_t`

Reply:
Also I propose to change the name. The word `changes` is more for RTPS layer. Not for DDS API. Maybe an alternative is `clear_history()` o something similar.